static/js/src/navigation.js

```diff
@@ -30,18 +31,51 @@ navDropdowns.forEach(function (dropdown) {
       } else {
         dropdown.classList.remove("is-selected");
         dropdownContent.classList.add("u-hide");
+        dropdownContentMobile.classList.add("u-hide");
       }
     });
   });
 });
 
+var navigation = document.querySelector(".p-navigation");
+
+function mobileViewUpdate() {
+  var viewportWidth = window.innerWidth;
+  if (viewportWidth <= 1024) {
+    navDropdowns.forEach(function (dropdown) {
+      if (dropdown.classList.contains("is-selected")) {
+        navigation.classList.add("has-menu-open");
+      }
+    });
+  }
+}
+window.onload = mobileViewUpdate;
+window.onresize = mobileViewUpdate;
+
+window.addEventListener("test", function (e) {
+  var navigation = document.querySelector(".p-navigation");
+  function menuOpenMobile() {
+    navDropdowns.forEach(function (dropdown) {
+      if (
+        dropdown.classList.contains("is-selected") &&
+        window.innerWidth < 1024
+      ) {
+        navigation.classList.add("has-menu-open");
+      }
+    });
+  }
+  window.onresize = menuOpenMobile;
```

Comment:
This is likely to produce performance issues. Can we not just use CSS with media queries?

Reply:
When the screen gets smaller while the menu is open on big screen, the menu needs to be open on small screen but I couldn't find a way to open the menu with media queries.
![ezgif com-gif-maker (5)](https://user-images.githubusercontent.com/57550290/169297927-04c5061d-1c52-416f-a32a-a8716ce8d5c0.gif)

Reply:
Ok I see the issue. I think this could be improved in the future